sat/cli/bootsys/platform.py

```diff
@@ -484,7 +484,19 @@ def do_recreate_cronjobs(_):
     """Recreate cronjobs that are not being scheduled on time."""
     try:
         batch_api = load_kube_api(api_cls=BatchV1Api)
-        recreate_namespaced_stuck_cronjobs(batch_api, 'services')
+        # Adding a retry
+        max_retries = 3
+        retries = 0
+        while retries < max_retries:
+            try:
+                recreate_namespaced_stuck_cronjobs(batch_api, 'services')
+                break  # Exit the loop if successful
+            except Exception as e:
+                LOGGER.warning(f'Retrying due to Error: {e}')
```

Comment:
I would suggest saying that we are waiting 10 seconds before retrying, so the user is not confused by the lack of any progress for 10 seconds. You could even log a warning for the error, then an info message when the sleep is over saying it's trying again.